lib/web_ui/dev/integration_tests_manager.dart

```diff
@@ -47,18 +47,17 @@ class IntegrationTestsManager {
   }
 
   Future<bool> _runPubGet(String workingDirectory) async {
-    final String executable = isCirrus ? environment.pubExecutable : 'flutter';
-    final List<String> arguments = isCirrus
-        ? <String>[
-            'get',
-          ]
-        : <String>[
-            'pub',
-            'get',
-          ];
+    if (isCi) {
+      await _cloneFlutterRepo();
+    }
+    final String executable =
+        isCi ? environment.flutterCommandDir.path : 'flutter';
     final int exitCode = await runProcess(
       executable,
-      arguments,
+      <String>[
+        'pub',
```

Comment:
The more idiomatic command when fetching packages is `flutter packages get`. If we want to use `pub get`, then we don't need the Flutter checkout at all. We can just run `out/host_debug_unopt/dart-sdk/bin/pub get` (which will also drastically simplify this code).

Reply:
It is not correct.

We have flutter application we write in the integration test directories.  If we run pub get instead of flutter pub get we will get the following error:

```
Resolving dependencies... 
Because regular_integration_tests depends on e2e 0.4.0 which requires the Flutter SDK, version solving failed.

Flutter users should run `flutter pub get` instead of `pub get`.
```

Reply:
Ah, makes sense.